llm/README.md

```diff
@@ -8,4 +8,12 @@ transformers==4.31.0
 
 To execute the script, set env var INTEGRATION_AUTH_TOKEN and run:
 
-    python3 summarize.py
\ No newline at end of file
+    python3 summarize.py
+
+In order to run LLM models, we need to create the right AWS SageMaker Notebook Instance (with at least one GPU, and need to be compatible with the platform identifier).
+
+ml.p3.2xlarge is such instance with the lowest cost.
+
+See https://docs.aws.amazon.com/sagemaker/latest/dg/notebooks-available-instance-types.html for more instance information and https://aws.amazon.com/sagemaker/pricing/ for cost information.
+
+Make sure to request for AWS quota when choosing ml.p3.2xlarge if you are asked to do so.
```

Comment:
Can you describe detailed instructions on how to get it running step by step?

Reply:
Please do the same for setting up organization accounts, in a separate md file.

Reply:
Provided specific instructions for both.